readme/manual-setup.md

```diff
@@ -5,18 +5,22 @@ Welcome to the Ozone FOSS manual setup guide. This guide details the setup of Oz
 ### Create your working directory
 
 Move to the location of your choice, e.g., your home folder:
+
+Note: On Linux OS you'll have to open the terminal as root/admin user by typing `sudo -i` and then entering correct user's password if prompted
```

Comment:
@kdaud put this at the moment when it is really necessary. When exactly does the user start needing to be elevated with `sudo`?

Reply:
The docker compose fails the run for normal users(Here are samples of [logs](https://talk.openmrs.org/t/ozone-installation/38770/4), so the user must be elevated with privileges.

Reply:
From what have tested out, I would recommend we use `sudo -s` not `sudo -i`. I would need @Ruhanga's advice on this before I go a head to update the PR.

Reply:
> When exactly does the user start needing to be elevated with sudo?

At the start of the process after creating the working directory

Reply:
The ideal of elevating the user with `sudo` applies for both the Quick start guide and the manual setup guide.

Reply:
@kdaud, `sudo -s` is okay. Could you go ahead to add that change...